crates/sui/src/client_commands.rs

```diff
@@ -698,11 +698,19 @@ impl SuiClientCommands {
             }
             SuiClientCommands::Addresses => {
                 let active_address = context.active_address()?;
-                let addresses = context.config.keystore.addresses();
-                SuiClientCommandResult::Addresses(AddressesOutput {
-                    addresses,
+                let addresses = context
+                    .config
+                    .keystore
+                    .addresses_with_alias()
+                    .into_iter()
+                    .map(|(address, alias)| (alias.alias.to_string(), *address))
+                    .collect::<Vec<_>>();
```

Comment:
nit, I think the type annotation on `collect` is unnecessary here, but if it is necessary, it's slightly nicer to annotate the variable rather than use the turbofish:

```suggestion
                let addresses: Vec<_> = context
                    .config
                    .keystore
                    .addresses_with_alias()
                    .into_iter()
                    .map(|(address, alias)| (alias.alias.to_string(), *address))
                    .collect();
```